index.html

```diff
@@ -749,38 +518,44 @@
             ],
             initialMessage:
                 'Have fun.'
-        });
-    };
-
-    openSwitch.addEventListener('click', open, false);
-});
-</script>
-<script type="text/javascript">
-require(['explaingit'], function (explainGit) {
-    var openSwitch = document.getElementById('open-zen'),
-        open;
-
-    open = function () {
-        var elements = document.getElementsByClassName('row');
-        for(var i = 0; i != elements.length; ++i)
-        {
-          elements[i].style.display = 'none';
         }
-        document.getElementById('fork-me').style.display = 'none';
-        explainGit.reset();
-
-        explainGit.open({
-            name: 'Zen',
-            height: '100%',
-            commitData: [
-                {id: 'e137e9b', tags: ['master']}
-            ],
-            initialMessage:
-                'Have fun.'
-        });
     };
 
-    openSwitch.addEventListener('click', open, false);
+    window.addEventListener("hashchange", open, false);
+    window.addEventListener("load", open, false);
```

Comment:
It's been a while, but I think I've had every string only use single quotes in this entire codebase?